client/scripts/views/pages/view_proposal/create_comment.ts

```diff
@@ -163,12 +170,19 @@ const CreateComment: m.Component<{
               },
               tabindex: vnode.attrs.tabindex,
             }),
+            m('.token-requirement', [
+              tokenPostingThreshold && tokenPostingThreshold.gt(new BN(0))
```

Comment:
I'm assuming this will also need to check if Token community, bc otherwise it's not necessary to gate user behavior.